filebeat/tests/system/test_unix.py

```diff
@@ -16,25 +16,43 @@ class Test(BaseTest):
     Test filebeat UNIX input
     """
 
-    def test_unix_with_newline_delimiter(self):
+    def test_unix_with_newline_delimiter_over_stream_socket(self):
         """
         Test UNIX input with a new line delimiter
         """
-        self.send_events_with_delimiter("\n")
+        self.send_events_with_delimiter("\n", "stream", send_stream_socket)
 
-    def test_unix_with_custom_char_delimiter(self):
+    def test_unix_with_custom_char_delimiter_over_stream_socket(self):
         """
         Test UNIX input with a custom single char delimiter
         """
-        self.send_events_with_delimiter(";")
+        self.send_events_with_delimiter(";", "stream", send_stream_socket)
 
-    def test_unix_with_custom_word_delimiter(self):
+    def test_unix_with_custom_word_delimiter_over_stream_socket(self):
         """
         Test UNIX input with a custom single char delimiter
         """
-        self.send_events_with_delimiter("<END>")
+        self.send_events_with_delimiter("<END>", "stream", send_stream_socket)
 
-    def send_events_with_delimiter(self, delimiter):
+    def test_unix_with_newline_delimiter_over_datagram_socket(self):
+        """
+        Test UNIX input with a new line delimiter
+        """
+        self.send_events_with_delimiter("\n", "stream", send_stream_socket)
+
+    def test_unix_with_custom_char_delimiter_over_datagram_socket(self):
+        """
+        Test UNIX input with a custom single char delimiter
+        """
+        self.send_events_with_delimiter(";", "stream", send_stream_socket)
+
+    def test_unix_with_custom_word_delimiter_over_datagram_socket(self):
+        """
+        Test UNIX input with a custom single char delimiter
+        """
+        self.send_events_with_delimiter("<END>", "stream", send_stream_socket)
```

Comment:
unix datagram sockets do not have a splitter. The dgram packets is the complete message. I think we can remove these tests.